lib/apiservers/engine/backends/archive.go

```diff
@@ -37,6 +48,11 @@ import (
 	"github.com/docker/docker/pkg/archive"
 )
 
+type VicArchiveProxy interface {
+	ArchiveExportReader(ctx context.Context, store, ancestorStore, deviceID, ancestor string, data bool, filterSpec vicarchive.FilterSpec) (io.ReadCloser, error)
```

Comment:
might rename to ExportReader only? feels a bit duplicate with the interface name